InformationScripting/src/wrappers/DataApi.cpp

```diff
@@ -43,29 +43,49 @@ object Tuple_getAttr(const Tuple& self, const QString& name) {
 	return pythonObject(self[name]);
 }
 
+NamedProperty tuple_getItem(Tuple &t, int index)
+{
+	 if (index >= 0 && index < t.size())
+		  return t[index];
+	 else
+	 {
+		  PyErr_SetString(PyExc_IndexError, "index out of range");
+		  throw_error_already_set();
+	 }
+	 return {};
+}
+
 std::shared_ptr<Tuple> makeTuple(list args) {
 	stl_input_iterator<NamedProperty> begin(args), end;
 	return std::shared_ptr<Tuple>{new Tuple{QList<NamedProperty>::fromStdList(std::list<NamedProperty>(begin, end))}};
 }
 
 BOOST_PYTHON_MODULE(DataApi) {
-		class_<NamedProperty>("NamedProperty", init<QString, QString>())
+		// This class exposure is just a workaround to make the name property of NamedProperty work.
+		class_<QPair<QString, Property>, boost::noncopyable>("NamedPropertyBaseClass", no_init);
```

Comment:
Could you please elaborate the comment, in the code, to explain what the problem is?